config/jobs/kubernetes-security/genjobs.go

```diff
@@ -139,6 +139,13 @@ func convertJobToSecurityJob(j *config.Presubmit, dropLabels sets.String) {
 	j.Trigger = strings.Replace(j.Trigger, "pull-kubernetes", "pull-security-kubernetes", -1)
 	j.Context = strings.Replace(j.Context, "pull-kubernetes", "pull-security-kubernetes", -1)
 
+	// if a GKE job, disable it
+	if strings.Contains(j.Name, "gke") {
+		j.AlwaysRun = false
+		j.Optional = true
+		j.Trigger = ""
+	}
```

Comment:
just don't generate it?

Reply:
we have a bunch of other logic making sure we do generate them :-)

This is also minimally invasive, even if we get them working later this will keep them at manually triggered.

Reply:
ack

Reply:
done